src/resource.js

```diff
@@ -43,6 +43,7 @@ var Resource = function() {
   this.$$links = {};
   this.$$embedded = {};
   this.$$forms = {};
+  this.$$formatSpecific = {};
```

Comment:
The double $$ prefix is reserved for "private" properties. If we're not going to offer convenience functions for accessing the data in `$$formatSpecific`, then it probably should be renamed to `$formatSpecific` with a single $.